plugins/poll/spec/lib/poll_spec.rb

```diff
@@ -337,4 +338,227 @@
       )
     end
   end
+
+  describe ".serialized_voters" do
```

Comment:
I have added tests here to validate the fix, but I see that there are also offset tests in `plugins/poll/spec/controllers/polls_controller_spec.rb`. If required I can also add validations there.

Reply:
Btw, most of the time we favour repetition over `shared_examples`. 

For example in the shared example below there needs to be knowledge of `ranked_choice`, and the reader has to refer to and fro to the expected values.

Reply:
Indeed. I can edit the PR to remove the `shared_examples`.